frontend/src/metabase/query_builder/components/filters/pickers/RelativeDatePicker.jsx

```diff
@@ -46,7 +46,7 @@ export default class RelativeDatePicker extends Component {
 
   render() {
     const {
-      filter: [op, field, intervals, unit],
+      filter: [op, field, intervals, unit, options],
```

Comment:
The problem with the flow type above is you're destructuring `options` even if it's the 4 arity form of `TimeIntervalFilter`.